Gemfile.tt

```diff
@@ -2,16 +2,19 @@ source 'https://rubygems.org'
 ruby '<%= RUBY_VERSION %>'
 
 # Backend
-gem 'rails', '6.1.3.1' # Latest stable
+gem 'rails', '7.0.1' # Latest stable
 gem 'pg' # Use Postgresql as database
 gem 'puma' # Use Puma as the app server
 gem 'mini_magick' # A ruby wrapper for ImageMagick or GraphicsMagick command line
 gem 'pagy' # A pagination gem that is very light and fast
 gem 'discard' # Soft deletes for ActiveRecord
-gem 'ffaker' # A library for generating fake data such as names, addresses, and phone numbers.
-gem 'fabrication' # Fabrication generates objects in Ruby. Fabricators are schematics for your objects, and can be created as needed anywhere in your app or specs.
 gem 'sidekiq' # background processing for Ruby
 gem 'bootsnap', require: false # Reduces boot times through caching; required in config/boot.rb
+gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby] # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
+# gem 'jbuilder' # Build JSON APIs with ease
+# gem 'redis' # Use Redis adapter to run Action Cable in production
+# gem 'kredis' # Use Kredis to get higher-level data types in Redis
```

Comment:
Is that added by default? It feels a bit strange to have commented out dependencies 🤔

Reply:
Yup I brought this from the project that is generated from default rails template 🙌 

![Screen Shot 2565-01-12 at 16 47 12](https://user-images.githubusercontent.com/6965195/149116888-efae7735-9b03-4ba5-8c55-f7c822990c46.png)

I put comment because I'm not sure if we will use it or not (Same for `jbuilder`)

By putting comment on them, My idea is to help us to not forget to checkout those gems later if we want to include it or not. What do you think?

Reply:
Understood that it comes by default so it can be kept commented as done here